testserver/testserver.go

```diff
@@ -300,7 +316,7 @@ func NewTestServer(opts ...TestServerOpt) (TestServer, error) {
 		"--host=localhost",
 		"--port=0",
 		"--http-port=0",
-		"--store=" + baseDir,
+		storeArg,
```

Comment:
can you verify the logging behavior with an in-mem store? i think by default the logs go to the console, so we will want to change it so the logs are written to a file instead. i notice that the `logDir` variable is unused right now, so we should probably use it somewhere.


unfortunately the way logging is configured has changed in different versions

in 21.1, i think we use `--log="{file-defaults: {dir: /path/to/log-dir}, sinks: {stderr: {filter: NONE}}}"`
previously, we would use `--log-dir=...`